Microsoft.Toolkit.Uwp.SampleApp/SamplePages/RichSuggestBox/RichSuggestBoxPage.xaml.cs

```diff
@@ -91,13 +89,14 @@ public sealed partial class RichSuggestBoxPage : Page, IXamlRenderListener
 
         private RichSuggestBox _rsb;
         private RichSuggestBox _tsb;
-        private DispatcherQueueTimer _hoveringTimer;
+        private DispatcherQueue _dispatcherQueue;
         private PointerPoint _pointerPoint;
 
         public RichSuggestBoxPage()
         {
             this.InitializeComponent();
-            this._hoveringTimer = DispatcherQueue.GetForCurrentThread().CreateTimer();
+            DispatcherQueue.GetForCurrentThread().CreateTimer();
```

Comment:
This timer line doesn't do anything now, eh?

Reply:
Wow, interesting that Resharper doesn't delete the whole statement when I safe-delete the field. I will remove it in the next commit.